repos/keg-components/src/components/form/scrollable/select/scrollableSelect.stories.mdx

```diff
@@ -0,0 +1,31 @@
+<!--- scrollableSelect.stories.mdx -->
```

Comment:
I'm getting an error when I try to view this in StoryBook => http://snpy.in/XMACUc

Reply:
Hmm, not able to recreate this, but try the new image and let me know if you still encounter it.